### ISSUE TITLE
Stopping fluxion from using internal WIFI card when 2 packet injection capable cards available 

### ISSUE THREAD

**Issue by powerpcme** [wi-fi-analyzer/fluxion#30]
### Which version of fluxion are you using?
fluxion 2 rev. 9

### What distribution of Linux (including the version) are you running it on?
Kali Linux 2017.1 4.13.0-kali1-amd64

### What wireless adapter are you using? Exact model and chipset, statements like "internal" and "was working before" are not helpful at all.
Alfa AWUS036NH, TP Link TL-WN722N, Broadcom 4360

### Which is the driver for it?
Not sure

### Does it support injection ( output of aireplay-ng -9)
Both do except the Broadcom 

### General description of your issue along with the steps to reproduce it
Basically, I'd like to JUST use the Alfa and the TP Link but what seems to be happening both those cards are being used for deauthing and the internal is being used for the fake AP. I've tried using "ifconfig CARD down" but fluxion will still use that card. What can I do to disable the card before fluxion use?
